src/content/docs/en/reference/components-reference.mdx

```diff
@@ -0,0 +1,265 @@
+---
+title: Built-in Components Reference
+i18nReady: true
+tableOfContents:
+  minHeadingLevel: 2
+  maxHeadingLevel: 2
+---
+import Since from '~/components/Since.astro';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+import ReadMore from '~/components/ReadMore.astro';
+
+Astro includes several built-in components for you to use in your projects. All built-in components are available in `.astro` files via `import {} from 'astro:components'`.
```

Comment:
The heading that was here was redundant (pulled directly from the other page where it was simply a page section.)

This means we don't need it here, and all the components can be `##`.